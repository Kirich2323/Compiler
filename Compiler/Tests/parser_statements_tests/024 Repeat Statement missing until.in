var
    a : integer = 0;
begin
    repeat
        a := a + 10;
     a > 20
end.