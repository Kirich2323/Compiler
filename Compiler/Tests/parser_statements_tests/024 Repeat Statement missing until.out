(6 ; 6): Got unexpected token: "identifier". Expected: "until".